src/utils/cookieUtils.ts

```diff
@@ -0,0 +1,43 @@
+const includeDomain = process.env.NODE_ENV !== "test"
+
+export const getTimeLeft = (expTime) => {
+  console.log(expTime)
+  return (new Date(expTime).getTime() - new Date().getTime()) / 1000
+}
+/**
+ * setCookieWithMaxAge(sKey, expiration, value)
+ * set a cookie
+ * @param {string} sKey - The name of the cookie
+ * @param {string} expiration - when to expire the cookie. Defaults to immediately expire
+ * @param {string} value - The value of the cookie, defaults to an empty string
+ */
+
+const setCookieWithExpiration = (sKey, expiration, value = "") => {
```

Comment:
Where does the sKey name come from? Can this be renamed as just "key"?

Reply:
it was just like that in DFE